repose-aggregator/artifacts/extensions-filter-bundle/src/main/application/WEB-INF/web-fragment.xml

```diff
@@ -32,4 +32,8 @@
         <filter-name>saml-policy</filter-name>
         <filter-class>org.openrepose.filters.samlpolicy.SamlPolicyTranslationFilter</filter-class>
     </filter>
+    <filter>
+        <filter-name>attribute-mapping-policy-validation</filter-name>
```

Comment:
I have been told that this name is quite long. I am open to other suggestions, though I'm also fine leaving it as-is since it is only for one user.